doc/classes/NavigationMesh.xml

```diff
@@ -18,6 +18,12 @@
 				Adds a polygon using the indices of the vertices you get when calling [method get_vertices].
 			</description>
 		</method>
+		<method name="clear">
+			<return type="void" />
+			<description>
+				Clears the internal arrays, e.g. vertices and polygon indices.
```

Comment:
You probably meant to use `i.e.`? "e.g." means "for example", "i.e." means "that is to say".

Reply:
I think I made this weird. Should go change to something simple like ... "Clears the internal arrays for vertices and polygon indices."

Reply:
Yeah that sounds great.